src/core/DigitalTwinApi.ts

```diff
@@ -62,4 +62,13 @@ export class DigitalTwinApi implements Api {
         return await cogniteClient.post(`${cogniteClient.getBaseUrl()}/api/playground/projects/${cogniteClient.project}/twins/delete`, options)
     }
 
+    public retrieveJson = async (configId: number): Promise<HttpResponse<any>> => {
+        return await cogniteClient.get(`${cogniteClient.getBaseUrl()}/api/playground/projects/${cogniteClient.project}/twins/${configId}`)
+            .then(response => {
```

Comment:
Is it required to add both `await` and `then`

Reply:
This function also from Mihil's changes that should revert and I have missed it

Reply:
fix added